portal/components/tunnelStatusUpdaters/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { BitcoinDepositsStatusUpdater } from './bitcoinDepositsStatusUpdater'
+import { BitcoinWithdrawalsStatusUpdater } from './bitcoinWithdrawalsStatusUpdater'
+import { EvmDepositsStatusUpdater } from './evmDepositsStatusUpdater'
+import { EvmWithdrawalsStateUpdater } from './evmWithdrawalsStateUpdater'
+
+export const TunnelStatusUpdaters = () => (
+  <>
+    {/* Track updates on bitcoin deposits, in bitcoin or in Hemi */}
+    {<BitcoinDepositsStatusUpdater />}
```

Comment:
Extra curly braces?

Reply:
🤦🏽  copy-paste error. Removed in [c72c4c1](https://github.com/hemilabs/ui-monorepo/pull/1217/commits/c72c4c13bd098ea74f2d283c19dae97c43159a7d)